lib/obanalyze/oban_jobs.ex

```diff
@@ -59,7 +59,7 @@ defmodule Obanalyze.ObanJobs do
 
     from oj in query,
       where: like(oj.worker, ^like),
-      or_where: like(oj.args, ^like)
+      or_where: like(fragment("?::text", oj.args), ^like)
```

Comment:
I think this is postgres way of casting and sqlite doesn't do it like this. Try the explicit `cast()` function that both DBs support.